Documentation/configuration/sbom.rst

```diff
@@ -31,22 +31,32 @@ Prerequisites
 Download SBOM
 =============
 
-The SBOM can be downloaded from the supplied Cilium image using the
-``cosign download sbom`` command.
+You can download the SBOM in-toto attestation from the supplied Cilium image using the following command:
 
 .. code-block:: shell-session
 
-    $ cosign download sbom --output-file sbom.spdx <Image URL>
+    $ cosign download attestation --predicate-type spdxjson <Image URI> | jq -r .payload | base64 -d | jq .predicate > ciliumSBOM.spdx.json
 
-Verify SBOM Image Signature
-===========================
+Verify SBOM attestation
+=======================
 
-To ensure the SBOM is tamper-proof, its signature can be verified using the
-``cosign verify`` command.
+To verify the SBOM in-toto attestation on the supplied Cilium image, run the following command:
 
 .. code-block:: shell-session
 
-    $ COSIGN_EXPERIMENTAL=1 cosign verify --certificate-github-workflow-repository cilium/cilium --certificate-oidc-issuer https://token.actions.githubusercontent.com --attachment sbom <Image URL> | jq
+    $ TAG=v1.15.0
+    $ cosign verify-attestation --certificate-github-workflow-repository cilium/cilium \
+    --certificate-oidc-issuer https://token.actions.githubusercontent.com \
+    --certificate-identity "https://github.com/cilium/cilium/.github/workflows/build-images-releases.yaml@refs/tags/${TAG}" \
+    --type spdxjson <Image URI> | jq
```

Comment:
The json output is too big, can we add `  2>&1   | head -n 13` instead of `| jq`?

For example:

```
--type spdxjson quay.io/cilium/cilium-ci:e5495c548abd25b6672431c02bb6c0460925baff  2>&1   | head -n 13

Verification for quay.io/cilium/cilium-ci:e5495c548abd25b6672431c02bb6c0460925baff --
The following checks were performed on each of these signatures:
  - The cosign claims were validated
  - Existence of the claims in the transparency log was verified offline
  - The code-signing certificate was verified using trusted certificate authority certificates
Certificate subject: https://github.com/cilium/cilium/.github/workflows/build-images-ci.yaml@refs/pull/31028/merge
Certificate issuer URL: https://token.actions.githubusercontent.com
GitHub Workflow Trigger: pull_request
GitHub Workflow SHA: 89175f13a4a0d6b79e06902e3e6a6ab55358001a
GitHub Workflow Name: Image CI Build
GitHub Workflow Repository: cilium/cilium
GitHub Workflow Ref: refs/pull/31028/merge
```